views/FanView/FanView.tsx

```diff
@@ -16,6 +16,8 @@ import { styles } from './FanView.styles';
 
 type FanViewProps = NativeStackScreenProps<RootStackParamList, 'Fan'>;
 
+const numberRegex = new RegExp('\\.', 'g');
```

Comment:
Do we need this regex? I think that setting `keyboardType` as `numeric` is sufficient security